packages/data-transformation/fhir-to-csv/src/parseFhir/configurations/config_Medication.ini

```diff
@@ -109,25 +109,6 @@ ingredient_1_strength_numerator_code = ingredient.1.strength.numerator.code
 ingredient_1_strength_numerator_system = ingredient.1.strength.numerator.system
 ingredient_1_strength_numerator_unit = ingredient.1.strength.numerator.unit
 ingredient_1_strength_numerator_value = ingredient.1.strength.numerator.value
-ingredient_2_batch_expirationdate = ingredient.2.batch.expirationDate
-ingredient_2_batch_lotnumber = ingredient.2.batch.lotNumber
-ingredient_2_isactive = ingredient.2.isActive
-ingredient_2_itemcodeableconcept_coding_0_code = ingredient.2.itemCodeableConcept.coding.0.code
-ingredient_2_itemcodeableconcept_coding_0_display = ingredient.2.itemCodeableConcept.coding.0.display
```

Comment:
ditto: ingredient is codeableconcept, not coding, so should we keep more around?